sympy/integrals/integrals.py

```diff
@@ -832,6 +834,24 @@ def _eval_integral(self, f, x, meijerg=None, risch=None, manual=None,
             if not meijerg:
                 # g(x) = Mul(trig)
                 h = trigintegrate(g, x, conds=conds)
+                if isinstance(f,Sum):
+                    f_temp =f
+                    f, limits = f.function, list(f.limits)
+                    limit = limits.pop(-1)
```

Comment:
`limits ` is of type ` list ` with value `[(n, 1, N)]`, for further step type` tuple` is needed and that is type of `limit` with value `(n, 1, N)`.